functions/test/produce.test.js

```diff
@@ -0,0 +1,81 @@
+const expect = require("expect");
+const request = require("supertest");
+
+const app = require("../index");
+const test = require("firebase-functions-test")();
+const { produce } = require("./data");
+
+let id;
+
+describe("POST /produce", () => {
+  it("create a new produce", (done) => {
+    request(app)
+      .post("/produce")
+      .send(produce)
+      .expect((res) => {
+        expect(res.text).not.toBeNull();
```

Comment:
Can you expect anything beyond not null? Perhaps not.

Reply:
Because the document ID is added to the response when the document is created, I think that the expected response should be not null.

Reply:
Yeah, I meant more like can you expect that the text contains a certain word, or the response has something other than text, etc.